db/migrate/20150916093231_correct_publishing_app.rb

```diff
@@ -0,0 +1,9 @@
+class CorrectPublishingApp < Mongoid::Migration
+  def self.up
+    Artefact.where(owning_app: 'publisher', kind: 'smart-answer').update_all(owning_app: 'smartanswers')
+    Artefact.where(owning_app: 'publisher', kind: 'travel-advice').update_all(owning_app: 'travel-advice-publisher')
```

Comment:
We really don't want to be doing this.  I suspect that these incorrectly owned travel-advice artefacts should probably be removed.